### DISCUSSION TITLE
Intergrate with `null-ls` using `mason-null-ls`

### DISCUSSION THREAD

**Discussion by nichtsam** [VonHeikemen/lsp-zero.nvim#120] [Q&A]
I've tried out the instruction in [`advance-usage.md`](https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md#intergrate-with-null-ls),  but the null-ls didn't get attached.
I investigated and play around for a bit, I found that I also need to add `require("null-ls").setup()` along with it, and then everything works.

I'm quite new at neovim and all the eco system, so I wonder if I misunderstood something or is the instruction missing a step?

just for reference, this is my nvim setup with working null-ls: https://github.com/nichtsam/nvim

**Comment by VonHeikemen** [comment]
The instruction is probably missing that step. I'll fix that later.
